examples/src/pages/Viewer.tsx

```diff
@@ -98,6 +98,10 @@ export function Viewer() {
         continuousModelStreaming: true,
         pointCloudEffects: {
           pointBlending: (urlParams.get('pointBlending') === 'true' ?? undefined)
+        },
```

Comment:
Changed it for tests, because previously it was aligned with coordinates of domElement, so bugs similar to this one were harder to spot.

Reply:
Think we should keep it top left, and then deal with correctness in visual test.